docs/user_guide.rst

```diff
@@ -1,28 +1,129 @@
 User Guide
 ==========
 
-Basic example
--------------
+Overview
+--------
 
 tox is an environment orchestrator. Use it to define how to setup and execute various tools on your projects. The
-tool can be:
+tool can set up environments for and invoke:
+
+- test runners (such as :pypi:`pytest`),
+- linters (e.g., :pypi:`flake8`),
+- formatters (for example :pypi:`black` or :pypi:`isort`),
+- documentation generators (e.g., :pypi:`Sphinx`),
+- build and publishing tools (e.g., :pypi:`build` with :pypi:`twine`),
+- ...
+
+Configuration
+-------------
+
+*tox* needs a configuration file where you define what tools you need to run and how to provision a test environment for
+these. The canonical file for this is the ``tox.ini`` file. For example:
+
+.. code-block:: ini
+
+    [tox]
+    min_version = 4.2
```

Comment:
This is only here for backward compatibility, I prefer using `requires = tox>=4` instead.